src/Infrastructure.EntityFramework/AdminConsole/Models/Organization.cs

```diff
@@ -9,6 +9,18 @@ namespace Bit.Infrastructure.EntityFramework.AdminConsole.Models;
 
 public class Organization : Core.AdminConsole.Entities.Organization
 {
+    // Deprecated by https://bitwarden.atlassian.net/browse/PM-10863. This
+    // was replaced with `LimitCollectionCreation` and
+    // `LimitCollectionDeletion`.
+    public bool LimitCollectionCreationDeletion
+    {
+        get => LimitCollectionCreation || LimitCollectionDeletion;
+        set
+        {
+            LimitCollectionCreation = value;
+            LimitCollectionDeletion = value;
+        }
+    }
```

Comment:
Why is this still needed? If it's just to keep it in the db for another cycle, I think a shadow property is preferable so that it's not present on the class at all.

Reply:
A shadow property still has to go in the EF class in some capacity, right? I can simply it to a simple `get; set;`, and have done so on the commit below. Removing it completely generates an EF migration that removes the column, something which could pollute other folk's migrations if the property is removed without generating one.

Turned into a `get; set;` on https://github.com/bitwarden/server/pull/4809/commits/2e2752407c690db513e77bcc7a865c977e982b0b